220104-Added note on access shared secrets in Anypoint Secrets Manager (#58)
mulesoft/docs-functional-monitoring: modules/ROOT/pages/bat-bdd-reference.adoc

In your tests, you can access shared secrets in Anypoint Secrets Manager, which enables you to securely store sensitive information, such as a password, authentication token, endpoint URL, or webhook URL.

> Note: Currently only above mentioned shared resources we can access from Anypoint Secrets Manager, other resources are not supported (Eg: Certificate, etc).

Reviewer: ```suggestion
NOTE: Anypoint Secrets Manager enables you to access only the above-mentioned shared resources. Other resources are not accessible.
```
